clients/src/main/java/org/apache/kafka/common/ApiKey.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common;
+
+import org.apache.kafka.common.errors.UnsupportedVersionException;
+import org.apache.kafka.common.protocol.Protocol;
+
+/**
+ * Identifiers for Kafka APIs.
+ *
+ * This is a public API for both Kafka clients and internals.  Metadata which is not intended to be
+ * exposed to end-users should go in #{org.apache.kafka.common.protocol.ApiKeys} rather than here.
```

Comment:
Instructions for ourselves should be in a non-javadoc comment after the Javadoc.